ci/ci/github.py

```diff
@@ -277,6 +295,8 @@ def from_gh_json(gh_json, target_branch):
             {user['login'] for user in gh_json['requested_reviewers']},
             {label['name'] for label in gh_json['labels']},
         )
+        pr.increment_pr_metric()
```

Comment:
What about when a PR is merged and forgotten about?

Reply:
Shouldn't there be a matching decrement?

Reply:
yup, there should be. Updated to decrement any tracked pr that is not seen in a new github update and subsequently dropped.